codeowners/src/codeowners.rs

```diff
@@ -15,6 +20,42 @@ pub struct CodeOwners {
     pub owners: Option<Owners>,
 }
 
+// NOTE(Tyler): Presently, with wasm we only check for presence of codeowners object. Custom conversion
+// is needed to fully support parsing it.
```

Comment:
We need to implement a lot more traits if we want to support this. Not a priority/nor bandwidth for this atm. I did evaluate that it correctly checks for TS truthiness/falseness.